public/graph.js

```diff
@@ -361,6 +361,37 @@ class GraphController {
                 )
                 .attr("width", Width / Object.keys(dataFilteredWeek).length);
 
+            // Add tooltip for the total received sms graph
```

Comment:
We don't need tooltips for 10 mins view let us have it in per day only and add this if requested by the team.

Reply:
removed tooltips for 10 minute view

Reply:
Thanks